doc/en/components/grids/_shared/column-resizing.md

```diff
@@ -415,7 +415,87 @@ This approach is more performance optimized than auto-sizing post initialization
 
 `sample="/{ComponentSample}/column-auto-sizing", height="550", alt="{Platform} {ComponentTitle} Column Resizing Example"`
 
+<!-- WebComponents, Blazor -->
 
+## Styling
```

Comment:
Also there's a second styling section from the base grid PR below the one you have added, so these are now duplicate sections:

![image](https://github.com/IgniteUI/igniteui-xplat-docs/assets/10397980/94e6093c-838f-4fab-9294-175bd2c01030)